apps/volume-storage.html.md.erb

```diff
@@ -0,0 +1,135 @@
+</div>
+
+Explore further options for data storage in [Databases & Storage](/docs/database-storage-guides/).
+
+Fly Postgres has [its own usage docs](/docs/postgres/), so here we'll focus on using Fly Volumes with your app.
+
+## Launch an app with a Fly Volume
+
+**The first rule of Fly Volumes is: always run at least two of them per application.** The way to do this is to run two Machines. Volumes don't sync up by themselves; different apps will have their own ways of dealing with this, so we won't get into that here.
+
+### Launch the app, but don't deploy immediately.
+
+The app has to exist for the volume to be created. The volume has to exist before you can mount it to a Machine. The first deployment will create a Machine. So the shortest path to a Machine with a mounted volume begins by launching the app, and saying `N` to "deploy now?":
+
+```cmd
+fly launch 
+```
+```out
+...
+Wrote config file fly.toml
+? Would you like to deploy now? No
+Your app is ready! Deploy with `flyctl deploy`
+```
+
+### Configure the app to mount the volume
+
+Make sure there's a `[mounts]` section in `fly.toml`. As an example, the following configures the app to expose data from a volume named `myapp_data` under the `/data` directory of the application.
+
+```toml
+[mounts]
+source="myapp_data"
+destination="/data"
+```
+
+### Provision the volume
+
+Create a volume for the app, with the name you chose, in the same region you're deploying the app to:
+
+```cmd
+fly volumes create myapp_data --region lhr --size 1 --app myapp
+```
+
+### Deploy the app
+
+```cmd
+fly deploy 
+```
+
+On the first deployment, you'll get one Machine. You can confirm this with `fly status`.
+
+### Confirm the volume is mounted
+
+You can check on all the volumes in your app using `fly volumes list`. The `ATTACHED VM` column lets you know which Machine, if any, the volume is mounted on.
+
+```cmd
+fly volumes list
+```
+```out
+ID                      STATE   NAME    SIZE    REGION  ZONE    ENCRYPTED       ATTACHED VM     CREATED AT    
+vol_zmjnv8m81p5rywgx    created data    1GB     lhr     b6a7    true            5683606c41098e  3 minutes ago
+```
+
+You can also see the volume in the Machine's file system (there's only one Machine so far):
+```cmd
+fly ssh console -s -C df
+```
+```out
+? Select VM: lhr: 5683606c41098e fdaa:0:3b99:a7b:7e:3155:9844:2 nameless-feather-6339
+Connecting to fdaa:0:3b99:a7b:7e:3155:9844:2... complete
+Filesystem     1K-blocks   Used Available Use% Mounted on
+devtmpfs          103068      0    103068   0% /dev
+/dev/vda         8191416 172748   7582856   3% /
+shm               113224      0    113224   0% /dev/shm
+tmpfs             113224      0    113224   0% /sys/fs/cgroup
+/dev/vdb         1011672   2564    940500   1% /storage
```

Comment:
When I deployed with mount destination = "/data", as I copy pasted from line 56 above,   I was only able to see "/data" and not "/storage", likeso:
<img width="419" alt="image" src="https://user-images.githubusercontent.com/33826441/224145610-f03ec439-f153-4e57-9314-179ba8eb26ce.png">

Reply:
Eagle eyes! Fixed.